### PR TITLE
editing rush events and changed words on some forms

### PR DESCRIPTION
**Pull request by jonahanderson**
Allows editing for rush events if you are admin. all aspects on the form are editable, including "new rushees allowed".

also changed some words on forms from "save changes" to things like "create announcement", "create event", etc. Editing forms keeps the "save changes" button.

### CODE REVIEW

File: rush/templates/rush/events.html [hunk 1]
```diff
@@ -14,6 +23,72 @@ <h2>Rush Events</h2>
...
+                                    <div class="form-group row">
+                                        <label for="new_time" class="col-sm-2 col-form-label">Time</label>
+                                        <div class="col-sm-10">
+                                            <input type="time" class="form-control" id="edit_time_{{ event.id }}" name="time" placeholder="HH:mm:ss in 24-hour time" value="{{ event.time|date:'H:i' }}" required>
+                                        </div>
+                                    </div>
+                                    <div class="form-group row">
+                                        <label for="new_location" class="col-sm-2 col-form-label">Location</label>
+                                        <div class="col-sm-10">
+                                            <input type="text" class="form-control" id="edit_location_{{ event.id }}" name="location" value="{{ event.location }}" required>
```
Comment (csyager) [inline rush/templates/rush/events.html:73]: In the create form location is not required.  If we want to make it required, we should update it in the create form and the models as well.  I think I would vote it shouldn't be required, in which case we just need to remove the `required` property on this line

**Review by csyager — CHANGES_REQUESTED** [review]
One small comment, otherwise LGTM

File: rush/templates/rush/events.html (hunk 1 above)
Comment (jonahanderson) [reply rush/templates/rush/events.html:73]: good catch, agreed

**Review by csyager — APPROVED** [review]
LGTM

**Pull request merged**